### PR TITLE
Iteration7 host my events

### PR DESCRIPTION
**Pull request by pattarinn**

### CODE REVIEW

**Review by KongtappV — APPROVED** [review]
Good Code

File: eve_holder/tests/test_html.py [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+        url = reverse('eve_holder:event_cancel', args=(self.event1_id,))
+        response = self.client.get(url)
+        self.assertTemplateUsed(response, 'eve_holder/events/event_cancel.html')
+
+    # def test_render_visitor_close_notification_redirect_registered_page(self):
+    #     """Should render eve_holder/visitors/visitor_registered_events.html"""
+    #     self.client.login(username='visitor', password='testPassword')
+    #     url = reverse('eve_holder:close_notification', args=(self.event1_id,))
+    #     response = self.client.get(url)
+    #     self.assertTemplateUsed(response, 'eve_holder/visitors/visitor_registered_events.html')
```
Comment (james31366) [inline eve_holder/tests/test_html.py:84]: Can you delete this comment?

Comment (pattarinn) [reply eve_holder/tests/test_html.py]: Already removed.

**Review by james31366 — APPROVED** [review]
Thx for write testing code.

**Pull request merged**